salesagility/SuiteCRM
# Fix #9865 -  Tabs with validation error are not highlighted

- resolve #9865
## Description
When trying to save a record from any module, which contains validations in tabs other than the first, the tabs are  highlighted so that the user can easily find the error.

To do this, the javascript code that provides this functionality and that previously depended on the YUI library, now obsolete, is modified.


## Motivation and Context
The objective is to facilitate the detection of errors and improve the user experience.

## How To Test
1) Create a record in a module whose edit view contains more than one tab and has some required field in a different tab than the first one.
2) Save the record. Check that the save is locked, but the affected tabs are highlighted



### Final checklist


(It would be nice to specify in the documentation that the inheritance of security groups is applied exclusively at record creation time.)
